src/view/PlantSelectionPanel.java

```diff
@@ -0,0 +1,29 @@
+package view;
+
+import java.awt.Dimension;
+import java.awt.FlowLayout;
+
+import javax.swing.JButton;
+import javax.swing.JPanel;
+
+public class PlantSelectionPanel {
+	private static final int MAX_PLANTS = 5;

```

Comment:
Where did you get this number from?

Reply:
Random set number. Milestone 3 wants various types of plants, so i just chose 5. We can change it to a more general variable and have a method assign the amount of plants we wish to make.